### PR TITLE
[IE TESTS] Update ConstantResultSubgraphTest

### PR DESCRIPTION
**Pull request by mandrono**
Issue: 49256

Links for internal validation in ticket

### CODE REVIEW

**Comment by postrational** [comment]
Please check if this unblocks the following tests:
  * onnx_model_sum_one_input
* onnx_model_softmax_0D
* onnx_model_shape
* onnx_model_eye_like

https://github.com/openvinotoolkit/openvino/blob/cc000e57e06bf62b7adb8f52914d4a03ff09914d/ngraph/test/runtime/ie/unit_test.manifest#L55-L65

**Comment by mandrono** [comment]
> Please check if this unblocks the following tests:
> 
> * onnx_model_sum_one_input
> * onnx_model_softmax_0D
> * onnx_model_shape
> * onnx_model_eye_like
> 
> https://github.com/openvinotoolkit/openvino/blob/cc000e57e06bf62b7adb8f52914d4a03ff09914d/ngraph/test/runtime/ie/unit_test.manifest#L55-L65

These tests were enabled:
+ onnx_model_sum_one_input
+ onnx_model_softmax_0D
+ onnx_model_eye_like

**Comment by mandrono** [comment]
@yury-intel @iefode @ilyachur review please

**Review by iefode — APPROVED** [review]
IE Test part LGTM

**Review by yury-intel — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Comment by mandrono** [comment]
@dmitry-gorokhov could you review please?

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp [hunk 1]
```diff
@@ -61,6 +59,10 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*decomposition1_batch=5_hidden_size=10_input_size=30_.*tanh.relu.*_clip=0_linear_before_reset=1.*_targetDevice=CPU_.*)",
         // TODO: Issue 47556. [NGraph] CTCGreedyDecoderSeqLen. Invalid type transformation i64 to i32.
         R"(.*(CTCGreedyDecoderSeqLenLayerTest).*(idxPRC=I64).*)",
+        // TODO: Issue: 48096
```
Comment (mbencer) [inline inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp:62]: The problem with changing original constant names was resolved in https://github.com/openvinotoolkit/openvino/pull/4143/files#diff-729914533c44221a8064a8afcbe95002092488741b5459f3d49ef4ba62c73f85

**Comment by azhogov** [comment]
CI passed: https://gitlab-icv.inn.intel.com/inference-engine/product-configs/merge_requests/1371

**Pull request merged**